src/app/screens/Onboard/ConnectRaspiBlitz/index.tsx

```diff
@@ -16,10 +16,17 @@ export default function ConnectRaspiBlitz() {
   const [loading, setLoading] = useState(false);
 
   function handleChange(event: React.ChangeEvent<HTMLInputElement>) {
-    setFormData({
-      ...formData,
-      [event.target.name]: `https://${event.target.value.trim()}:8080`,
-    });
+    if (event.target.name === "url") {
+      setFormData({
+        ...formData,
+        [event.target.name]: `https://${event.target.value.trim()}:8080`,
```

Comment:
maybe we also want to have the port in an input? can the user change the port in raspiblitz? - if not then it's fine.

Reply:
@MoritzKa do you know if the user can change the port?